sqlnexus/TextRowsets.xml

```diff
@@ -2235,6 +2235,28 @@
       <Column name="compatibility_level" type="RowsetImportEngine.IntColumn" />
       <Column name="query_plan_hash" type="RowsetImportEngine.VarBinaryColumn" length="8" />
       <Column name="is_forced_plan" type="RowsetImportEngine.IntColumn" />
+      <!-- 620 JAMGRIF 022024 -->
+      <Column name="force_failure_count" type="RowsetImportEngine.BigIntColumn" />
+      <Column name="last_force_failure_reason_desc"  type="RowsetImportEngine.VarCharColumn" length="256" />
+      <!-- 620 JAMGRIF 022024 2017+ only-->
+      <Column name="plan_forcing_type_desc"  type="RowsetImportEngine.VarCharColumn" length="120" />
+    </Rowset>
+
+    <!-- 620 JAMGRIF 022024 -->
```

Comment:
```suggestion
```

Reply:
Basically, rebuild your solution, do another commit and push it.

Reply:
Done.